vmm/src/default_syscalls.rs

```diff
@@ -252,6 +252,20 @@ pub fn default_context() -> Result<SeccompFilterContext, Error> {
                     ],
                 ),
             ),
+            (
+                libc::SYS_getrandom,
```

Comment:
I'm actually sad that we have to whitelist a syscall just for that sole rng use somewhere in `dumbo` (which doesn't even require crypto grade random numbers). I'd rather remote the `rand` crate dependency and use a small local helper function. I wrote a quick prototype in the `xor_rng` branch from my fork. Can you pls have a look? (it'll be super awkward if the syscall comes from something completely different :-s)

Reply:
Will do. Could a guest end up in a situation where `SYS_getrandom` is unavoidable?

Reply:
I really don't think so :-s If it does happen for some reason we'll deal with it then. Btw, that rng function from my branch was kinda linear, so I've updated it to something a bit more fancy.